release/_releases_json.sh

```diff
@@ -52,7 +52,10 @@ function _process_product {
 		tr -d "/" | \
 		uniq)
 	do
+		if [[ ! "{$product_version}" =~ (7\.[0-9]+\.[0-9]+\-u1(1[3-9]|[2-9][0-9]+|[0-9]{3,})) ]]
+		then
 		_process_product_version "${product_name}" "${product_version}"
```

Comment:
@pgydev, last requests about this pull:

- Shouldn't it have a tabulation here? I mean
```
		if [[ ! "{$product_version}" =~ (7\.[0-9]+\.[0-9]+\-u1(1[3-9]|[2-9][0-9]+|[0-9]{3,})) ]]
		then
				_process_product_version "${product_name}" "${product_version}"
		fi
```
Instead of
```
		if [[ ! "{$product_version}" =~ (7\.[0-9]+\.[0-9]+\-u1(1[3-9]|[2-9][0-9]+|[0-9]{3,})) ]]
		then
		_process_product_version "${product_name}" "${product_version}"
		fi
```
- Can you please adjust the commit message to be like `LPD-22075 Exclude internal releases above u113`? The idea is follow the actual pattern in place for existing commit messages, e.g. no symbols like `-` or `_` separating the words, and give a little more context about the change. :slightly_smiling_face: Please check an example of an expected commit message: https://github.com/liferay/liferay-docker/commit/fc67c4792cefc45a165bdf9b1a9e8b3cdec5e033.

Thanks in advance

Reply:
@natocesarrego,

- You're right, I added a tabulation there.

- I adjusted the commit, I will do like this next time too.

Thank you :)